runtime/rasdump/javadump.cpp

```diff
@@ -5149,6 +5153,70 @@ JavaCoreDumpWriter::writeCPUinfo(void)
 	return;
 }
 
+void
+JavaCoreDumpWriter::writeCgroupMetrics(void)
+{
+	OMRPORT_ACCESS_FROM_J9PORT(_PortLibrary);
+	BOOLEAN isCgroupSystemAvailable = omrsysinfo_cgroup_is_system_available();
+	if (isCgroupSystemAvailable) {
+		OMRCgroupEntry *entryhead = omrsysinfo_get_cgroup_subsystem_list();
+		OMRCgroupEntry *cgEntry = entryhead;
+		OMRCgroupEntry *nextEntry = NULL;
+		BOOLEAN inContainer = FALSE;
+		int32_t rc = 0;
+		char *tag = (char *)"2CICGRPINFO";
+		int32_t errorCode = 0;
+		inContainer = omrsysinfo_is_running_in_container(&errorCode);
```

Comment:
This is a CPP file so variables can be declared at first use

Reply:
removed `BOOLEAN inContainer = FALSE`, Declared `inContainer` at its first use